### PR TITLE
Modify the input method of bitrate. Bug fix

### PR DESCRIPTION
**Pull request by ZXfkSIE**
# Modification

- Change the input method (as well as the related code) of bit rate. Now bit rate can be input manually, instead of limited choices.

# Bug Fixes
- BUG : system tray icon will be permanently hided after clicking that icon for showing the hided dialog.
  - CAUSE: `m_hideIcon->hide()` is called after clicking the icon.
  - SOLUTION: Just don't hide it.
- BUG: the destructor of `Dialog` is not called while exiting the application.
  - CAUSE: Objects allocated on the heap will not be `delete`d automatically.
  - SOLUTION: `delete` it manually after `exec()` returns.
 - BUG: CMake scripts failed to link FFmpeg with MSVC.

### CODE REVIEW

**Review by dijo7 — APPROVED** [review]

**Review by dijo7 — APPROVED** [review]

File: QtScrcpy/dialog.cpp [hunk 1]
```diff
@@ -156,7 +152,19 @@ void Dialog::updateBootConfig(bool toView)
     if (toView) {
         UserBootConfig config = Config::getInstance().getUserBootConfig();
 
-        ui->bitRateBox->setCurrentIndex(config.bitRateIndex);
+        if(config.bitRate == 0)
```
Comment (barry-ran) [inline QtScrcpy/dialog.cpp:155]: if的风格参考代码中其它if的写法，保持一致吧

Comment (barry-ran) [reply QtScrcpy/dialog.cpp:155]: if分支必须用{}，左大括号和if在同一行

File: QtScrcpy/dialog.ui [hunk 2]
```diff
@@ -7,7 +7,7 @@
     <x>0</x>
     <y>0</y>
     <width>500</width>
-    <height>769</height>
+    <height>858</height>
```
Comment (barry-ran) [inline QtScrcpy/dialog.ui:10]: 这个高度先不要改了，在某些高分屏上显示不开

File: QtScrcpy/res/i18n/zh_CN.ts [hunk 3]
```diff
@@ -266,6 +266,18 @@
         <translatorcomment>安卓录屏程序隐藏在这！</translatorcomment>
         <translation>安卓录屏程序隐藏在这！</translation>
     </message>
+    <message>
+        <source>2</source>
+        <translation type="unfinished"></translation>
+    </message>
+    <message>
+        <source>Mbps</source>
```
Comment (barry-ran) [inline QtScrcpy/res/i18n/zh_CN.ts:274]: 在ui编辑器中把bitRateBox设置为不用翻译Mbps就不会出现在这里了

File: QtScrcpy/dialog.ui (hunk 2 above)
Comment (ZXfkSIE) [reply QtScrcpy/dialog.ui:10]: 这个不是我改的，是Linux开发环境下系统自动调节的

File: QtScrcpy/dialog.cpp (hunk 1 above)
Comment (ZXfkSIE) [reply QtScrcpy/dialog.cpp:155]: 已改

File: QtScrcpy/res/i18n/zh_CN.ts (hunk 3 above)
Comment (ZXfkSIE) [reply QtScrcpy/res/i18n/zh_CN.ts:274]: 已改

File: QtScrcpy/dialog.ui (hunk 2 above)
Comment (barry-ran) [reply QtScrcpy/dialog.ui:10]: 那你还原回来再提交

Comment (barry-ran) [reply QtScrcpy/dialog.ui:10]: 把这个修一下就没问题了我就合入了，记得rebase下哈，棒！

Comment (ZXfkSIE) [reply QtScrcpy/dialog.ui]: 整完了。原来高度增加的原因是我不小心拖了个Vertical Spacer到最底部

**Review by barry-ran — APPROVED** [review]

**Pull request merged**